libbeat/processors/config.go

```diff
@@ -125,8 +125,10 @@ func extractInt(unk interface{}) (uint64, error) {
 func extractString(unk interface{}) (string, error) {
 	switch s := unk.(type) {
 	case string:
-		return string(s), nil
+		return s, nil
+	case *string:
```

Comment:
@andrewkroh I think we need to tackle this problem in the event, and make sure that the event we are exporting doesn't contain any pointers. We shouldn't allow pointers in the condition of the processor.